reference/scripts.v3.yml

```diff
@@ -180,7 +180,8 @@ paths:
           in: query
           type: integer
         - name: sort
-          description: Field name to sort the scripts by.
+          description: |
+            Field name to sort the scripts by. Note: Since `id` increments when new scripts are added, you can use that field to sort by scripts create date.
```

Comment:
```suggestion
            Field name to sort the scripts by. Note: Since `id` increments when new scripts are added, you can use that field to sort by script create date.
```